### PR TITLE
added codewars

### LINKED ISSUE #911: [Add] Codewars
### Site URL

https://www.codewars.com/

### Description of the above provided site

Codewars serves as an online platform offering coding challenges suited for individuals across diverse programming proficiency levels.

### Category

Frontend

### Sub Category

Online Code Editors

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Bhanuteja01**
## Fixes Issue
closes #911 

## Changes proposed
added codewars site in frontend

### CODE REVIEW

**Comment by Bhanuteja01** [comment]
@CBID2  @rupali-codes 
waiting for review

**Review by rupali-codes — CHANGES_REQUESTED** [review]
your code doesn't follow the structure, please update it accordingly

**Comment by rupali-codes** [comment]
> I apologize for my mistake

Its fine,
You don't have to remove programimz, just add the resource right after that

**Comment by Bhanuteja01** [comment]
Ok 

> > I apologize for my mistake
> 
> Its fine, You don't have to remove programmiz, just add the resource right after that

****

File: database/frontend/online-code-editors.json [hunk 1]
```diff
@@ -89,5 +89,13 @@
     "url": "https://www.programiz.com/",
     "category": "frontend",
     "subcategory": "online-code-editors"
+  },
+  {
+
+    "name": "codewars",
+    "description": "Codewars serves as an online platform offering coding challenges suited for individuals across diverse programming proficiency levels.",
```
Comment (rupali-codes) [inline database/frontend/online-code-editors.json:96]:
```suggestion
    "description": "Codewars is an online platform that offers coding challenges suitable for individuals with diverse programming proficiency levels.",
```

**Review by rupali-codes — APPROVED** [review]
looks perfect, thanks

**Pull request merged**